src/components/Header/Header.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {HeaderWrapper, InputSearcher, IconWrapper} from './Header.styled';
+import logo from '../../logo-fur.jpg';
+import icon from '../../shopping-cart.png';
+import log from '../../user.png';
+
+
+
+
+export default function Header() {
+    return (
+        <HeaderWrapper>
+            <a href=".">
+                <img src={logo} alt="logo" className="img-logo"/>
+            </a>
+            <InputSearcher 
+                type="text" 
+                id ="search-box" 
+                className="input-search" 
+                placeholder=" Search..."/>
+            <IconWrapper>
+                <a href=".">
+                    <img src={log} alt="user-icon" className="user-icon"/>
+                </a>
```

Comment:
the icons have two responsabilities (according to the UI implemented): the first shows the profile section and the other one shows the ecommerce car. What do you think if the code implement two section. ProfileWrapper and CarEcommerceWrapper. Something like that. By this way the code add semantic and provide context. what du you think?